db/migrate/20230830115221_add_openid_scope.rb

```diff
@@ -0,0 +1,11 @@
+class AddOpenidScope < ActiveRecord::Migration[7.0]
+  def self.up
+    add_column :oauth_tokens, :allow_openid, :boolean, :null => false, :default => false
+    add_column :client_applications, :allow_openid, :boolean, :null => false, :default => false
```

Comment:
Why are we adding openid connect support to OAuth 1? That's not even a thing is it?

Reply:
Thanks for catching this. There is a number of classes used in both Oauth1 and Oauth2, for a moment I thought that these tables are used for both.

It was my (failed) attempt to fix the Oauth1 tests failing due to new `openid` scope in `lib/oauth.rb`. I will have to take into account that this scope is _only_ applicable to Oauth2,